### ISSUE TITLE
style files still bundled

### ISSUE THREAD

**Issue by derhuerst** [rtsao/csjs-extractify#4]
First of all: Thanks for this pretty elegant way of having CSS in JS!

I used `csjs-extractify` and the css files generated are correct. But the css is still inside the browserify bundle. Therefore the JS bundle will still be the same size (plus the generated CSS) and there will also be a runtime overhead.

This following behaviour is what i would have expected.

``` js
// foo.csjs.js
const csjs = require('csjs')
module.exports = csjs `
  .foo { color: red; }
  .bar { color: blue; }
`
```

``` js
// foo.csjs.js inside the JS bundle
module.exports = {
  foo: '.foo_12k4'
  bar: '.bar_ij5t'
`
```

**Comment by rtsao** [comment]
Yeah, this is a known issue and was one of the things brought up in https://github.com/rtsao/csjs-extractify/issues/3 but it's somewhat difficult to do statically at build-time. That said, what you are describing is definitely possible to do but I haven't had a chance to implement it yet 👍

**Comment by derhuerst** [comment]
I agree that it's not possible with static analysis in every possible way the code might look like. Once this is implemented, there should be a note on that in the readme that you can only access the class names after transpiring, not more.
